sqldelight-idea-plugin/build.gradle

```diff
@@ -9,7 +9,7 @@ intellij {
   version "IC-${versions.idea}"
   pluginName = 'SQLDelight'
   plugins = [
-      "org.jetbrains.kotlin:${versions.kotlin}-release-IJ${versions.idea}-1",
+      "org.jetbrains.kotlin:${versions.kotlin}-release-IJ2018.2-1",
```

Comment:
annoyingly - the kotlin plugin doesnt follow the same versioning as intellij. This works, and will fail when we upgrade to 2018.3 so whevs.